TrueCraft.API/Coordinates3D.cs

```diff
@@ -391,19 +180,27 @@ public static Coordinates3D Max(Coordinates3D value1, Coordinates3D value2)
             return new Coordinates3D(a % b.X, a % b.Y, a % b.Z);
         }
 
+        #endregion
+
+        #region Conversion operators
+
         public static explicit operator Coordinates3D(Coordinates2D a)
         {
             return new Coordinates3D(a.X, 0, a.Z);
         }
 
-        public static implicit operator Coordinates3D(Vector3 a)
+        public static explicit operator Coordinates3D(Vector3 a)
         {
-            return new Coordinates3D((int)a.X, (int)a.Y, (int)a.Z);
+            return new Coordinates3D((int)a.X,
+                                     (int)a.Y,
+                                     (int)a.Z);
         }
 
-        public static implicit operator Vector3(Coordinates3D a)
+        public static explicit operator Coordinates3D(Size s)
```

Comment:
Oh, I got the order wrong - you're right that it won't break anything. I agree with explicit when precision is lost.